assets/scss/btn.scss

```diff
@@ -0,0 +1,46 @@
+.btn {
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  cursor: pointer;
+  display: inline-block;
+  padding: .375rem .75rem;
+  transition: color .15s ease-in-out,background-color .15s ease-in-out,border-color .15s ease-in-out,box-shadow .15s ease-in-out;
+
+  &:active, &:focus, &:hover {
+    text-decoration: none;
+  }
+
+  &:hover {
+    background-color: #f9f9f9;
```

Comment:
I wasn't sure if the hover colours were used in places other than buttons.  I can add it, but it'd feel weird it was only used for buttons.

Also, `darken` is a dart-sass only function, and I'm hoping to avoid adding that dependency in light of recent Google layoffs and general dependency reduction.